pynuodb/encodedsession.py

```diff
@@ -101,10 +161,24 @@ def open_database(self, db_name, parameters, cp):
         self.putNull().putString(cp.genClientKey())
 
         self._exchangeMessages()
-        version = self.getInt()
+        protocolVersion = self.getInt()
         serverKey = self.getString()
         salt = self.getString()
-        return version, serverKey, salt
+        self.__connectionDatabaseUUID = self.getUUID()
+        
```

Comment:
Just to make a note of what we talked about. These variables are going to be kept as locals to the object instance, thus more than one active connection can be held without interference